.github/workflows/func.yaml

```diff
@@ -76,13 +76,18 @@ on:
         type: string
         default: ""
         description: "External juju controller name to use"
+      pytest-env:
+        required: false
+        type: string
+        description: "Extra pytest environment for pytest, base64 encoded env file"
```

Comment:
nit: This could just be: `Base64 encoded pytest-env file`

Reply:
The important part should be `env file` instead of `pytest-env`. `env file` is a proper noun.